src/core/components/operation.jsx

```diff
@@ -111,7 +111,7 @@ export default class Operation extends PureComponent {
     let onChangeKey = [ path, method ] // Used to add values to _this_ operation ( indexed by path and method )
 
     return (
-        <div className={deprecated ? "opblock opblock-deprecated" : isShown ? `opblock opblock-${method} is-open` : `opblock opblock-${method}`} id={escapeDeepLinkPath(isShownKey.join("-"))} >
+        <div className={(deprecated ? "opblock opblock-deprecated" : `opblock opblock-${method}`) + (isShown ? ` is-open` : "")} id={escapeDeepLinkPath(isShownKey.join("-"))} >
         <OperationSummary operationProps={operationProps} toggleShown={toggleShown} getComponent={getComponent} authActions={authActions} authSelectors={authSelectors} specPath={specPath} />
```

Comment:
The `is-open` class was not showing on the deprecated operations.

Reply:
**Change request**: use `classnames` instead of ternary operators here, example: https://github.com/swagger-api/swagger-ui/blob/ad786b023f8e54fcdb4a883039425148d8a8a327/src/core/json-schema-components.jsx#L258

it does have a lot of other usage patterns, see [here](https://github.com/JedWatson/classnames#usage) for documentation!